Add tasks on Bell states changes to Multi-Qubit States kata (#1385)
microsoft/qsharp: katas/content/multi_qubit_systems/bell_state_change_1/solution.md

We recognize that the goal is another Bell state. In fact, it is one of the four Bell states.

> We remember from "Sign flip" that the Pauli Z gate will change the state of the $|0\rangle$ basis state of a single qubit, so this gate seems like a good candidate for what we want to achieve. This gate leaves the sign of the $|0\rangle$ basis state of a superposition unchanged, but flips the sign of the $|1\rangle$ basis state of the superposition.

Reviewer: We need to take care with references here, since those tasks are in a different kata now. We can refer to the whole kata
```suggestion
We remember from the Single-Qubit Gates kata that the Pauli Z gate will change the state of the $|1\rangle$ basis state of a single qubit, so this gate seems like a good candidate for what we want to achieve. This gate leaves the sign of the $|0\rangle$ basis state of a superposition unchanged, but flips the sign of the $|1\rangle$ basis state of the superposition.
```

Don't forget that the Z gate acts on only a single qubit, and we have two here.

> Lets also remember how the Bell state is made up from its individual qubits.

Reviewer: ```suggestion
Let's also remember how the Bell state is made up from its individual qubits.
```

If the two qubits are A and B, where A is `qs[0]` and B is `qs[1]`, we can write that

> $|\Phi^{+}\rangle = \frac{1}{\sqrt{2}} \big(|0_{A}0_{B}\rangle + |1_{A}1_{B}\rangle\big)$, and also remember that this is a superposition.

Reviewer: I'm not sure what is the point of reminding that this is a superposition without an explanation, so let's remove this
```suggestion
$|\Phi^{+}\rangle = \frac{1}{\sqrt{2}} \big(|0_{A}0_{B}\rangle + |1_{A}1_{B}\rangle\big)$.
```

If we apply the Z gate to the qubit A, it will flip the phase of the basis state $|1_A\rangle$. As this phase is in a sense spread across the entangled state, with $|1_A\rangle$ basis state being part of the second half of the superposition, this application has the effect of flipping the sign of the whole basis state $|1_A1_B\rangle$, as you can see by running the solution below.

> > If you run the solution below several times, you will notice that you get the state $\frac{1}{\sqrt{2}} \big(-|00\rangle + |11\rangle\big)$ at a guess as frequently as the goal state. Here is why.

Reviewer: This should not be the case - once you've updated the testing harness, please check, and if it doesn't happen, let's remove this note
